.circleci/config.yml

```diff
@@ -2,14 +2,25 @@ version: 2
 
 references:
   container_config: &container_config
+    setup_remote_docker:
```

Comment:
I see [this is step](https://circleci.com/docs/2.0/configuration-reference/#setup_remote_docker) so unclear how this here.